test/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ list(APPEND tests
   stream-send-recv-ipv6
   stream-write-read
   stream-write-read-ipv6
+  stream-write-read-perf
```

Comment:
Let's mark this as disabled so we make sure it compiles as part of the build but doesn't grind the test suite to a halt. You can do it the same way I've done it here:

https://github.com/holepunchto/libjsc/blob/fe2f4b92e5ed6a3d455db4a561eb03777edf2797/test/CMakeLists.txt#L133-L139